packages/cypress/package.json

```diff
@@ -63,13 +63,13 @@
   },
   "dependencies": {
     "@segment/analytics-node": "^1.1.0",
-    "@storybook/csf": "^0.1.0",
-    "chrome-remote-interface": "^0.33.0",
-    "rrweb-snapshot": "^2.0.0-alpha.4"
-  },
-  "peerDependencies": {
     "@storybook/addon-essentials": "^7.0.0 || ^8.0.x",
+    "@storybook/csf": "^0.1.0",
     "@storybook/server-webpack5": "^7.0.0 || ^8.0.x",
+    "chrome-remote-interface": "^0.33.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
```

Comment:
Won't these be a problem if the project depending on this already has react installed?

Reply:
We only use React for the manager build, where the user's code is not involved, so, no I don't think it'll cause a problem (aside from React, a fairly large dependency) potentially being installed twice. We could make it a version range to (try to) avoid that.

Alternatively, if we lock down our version to SB8 only, I believe we can drop the react/-dom dependency here. We should figure out if we want to do that. The reason not to is to avoid installing the rest of SB twice where possible.

Reply:
Please go with SB8+ if you can. I suspect it will make everybody's lives a lot easier.